README.md

```diff
@@ -178,6 +179,49 @@ kubectl rook-ceph ceph versions
     }
 }
 ```
+### Destroy Cluster
```

Comment:
Let's revert this file. The readme is intended to give an overview of a few of the most commonly used commands. Let's just keep the destroy cluster in the separate doc since it's not commonly expected to be used.